### ISSUE TITLE
Error with Typo3 v10

### ISSUE THREAD

**Issue by ko1234** [web-vision/wv_deepltranslate#9]
If I used this with typo3 in version 10.4.13 i became an error:

`Core: Exception handler (WEB): Uncaught TYPO3 Exception: Too few arguments to function TYPO3\CMS\Backend\Controller\Page\LocalizationController::getPageColumns(), 1 passed in /var/www/html/public/typo3conf/ext/wv_deepltranslate/Classes/Override/LocalizationController.php on line 188 and exactly 3 expected`

This is the line:

```
return (new JsonResponse())->setPayload([
   'records' => $records,
   'columns' => $this->getPageColumns($pageId),
   ]);
```

If i edited it to this:

```
return (new JsonResponse())->setPayload([
   'records' => $records,
   'columns' => $this->getPageColumns($pageId, $records, []),
   ]);
```

it will work.
But it is not a good practice to work in the extension files.

**Comment by sypets** [comment]
Can you add how to reproduce this?

I wanted to test current version and the PR #12 and did not get any error with translation and TYPO3 10. (have to use Google translate currently, because DeepL API key acquisition still in progress)

**Comment by griesi007** [comment]
i see the same error in a fresh clean install 10.4.16 installation without any other extension or configuration

**Comment by jweiland** [comment]
Same error here, in a 10.4.15 installation. I just installed the extension, added DeepL API key and try to translate a page. After selecting translation mode in step 1 of translation, the BE remains in step 3.

After applying the code change, the translation works.

**Comment by sypets** [comment]
I don't get it. It works for me with latest version (and I don't get the error).

In any case though, the change above is identical with PR #12

**Comment by ayacoo** [comment]
Fresh install (10.4.17) with deepl api. https://github.com/web-vision/wv_deepltranslate/pull/12 is helpful :-)

**Comment by borishinzer** [comment]
Are some of you maybe running container extension? We recently got a notice, that the extension has some issues with container, while grid elements works.

**Comment by ayacoo** [comment]
Just a simple ddev installation and only this extension.

**Comment by NairCoder** [comment]
The issue is no more and fixed with #12 and is in master. Tested in TYPO3 v10.4.17
Thanks for the patience guys. :)
